### PR TITLE
Friedman test - Comp

### PR DESCRIPTION
**Pull request by feng-4chen**
Added R vs SAS code for Friedman test which is a nonparametric test for one-way repeated measures ANOVA on a simulated dataset. The code for the simulated data is also included.

### CODE REVIEW

File: R/friedman.qmd [hunk 1]
```diff
@@ -0,0 +1,43 @@
+## title: "friedman's test"
+
+### **Getting Started**
+
+Friedman's test is used when you have one within-subjects independent variable with two or more levels and a dependent variable that is not interval and normally distributed (but at least ordinal). To build such unreplicated blocked data, we'll create a data frame called `df_bp` from random number.
+
+```{r}
+#| eval: false
```
Comment (orladoylenvs) [inline R/friedman.qmd:8]: eval should be true to ensure you have imported tidyverse

File: R/friedman.qmd [hunk 2]
```diff
@@ -0,0 +1,43 @@
+## title: "friedman's test"
```
Comment (orladoylenvs) [inline R/friedman.qmd:1]: In a quarto file the the page should begin with the following format with just three dashes to denote the beginning and end:
# ---
# title: "Friedman's Test"
# ---

File: Comp/r-sas_friedman.qmd [hunk 3]
```diff
@@ -0,0 +1,105 @@
+------------------------------------------------------------------------
+
+title: "SAS vs R Non-parametric Analysis - Friedman test"
+
+------------------------------------------------------------------------
```
Comment (orladoylenvs) [inline Comp/r-sas_friedman.qmd:5]: In a quarto file the the page should begin with the following format with just three dashes to denote the beginning and end:

**Review by orladoylenvs — CHANGES_REQUESTED** [review]
Thanks for the submission. 

There are already R and SAS implementations for Friedman's - would you like to combine yours?
R version is here: https://github.com/PSIAIMS/CAMIS/blob/main/R/R_Friedmantest.qmd
SAS version is here: 
https://github.com/PSIAIMS/CAMIS/blob/main/SAS/SAS_Friedmantest.qmd

We don't have a SAS vs. R so that's a nice addition! Thank you!

**Comment by orladoylenvs** [comment]
Hi @feng-4chen  I think we still need to address the duplicates for Friedman in SAS and R? How would you like to proceed?

**Comment by feng-4chen** [comment]
> Hi @feng-4chen I think we still need to address the duplicates for Friedman in SAS and R? How would you like to proceed?

I worked with Sarfaraz together on Friedman. Maybe I could merge my SAS vs R compare in R file

**Comment by orladoylenvs** [comment]
Hi @feng-4chen, I'm not sure I am following. Apologies if I am repeating myself... 

For R: 
You contributed R/friendman.qmd and Safaraz contributed R/R_Friedmantest.qmd - these need to be merged into one file. For e.g. adding yours as an extra example. 

For SAS:
You contributed SAS/friedman.qmd and Safaraz contributed SAS/SAS_Friedmantest.qmd - these need to be merged into one file.  For e.g. adding yours as an extra example. 

For SAS vs R: 
You contributed Comp/r-sas_friedman.qmd and this is the only SAS vs R comparison so this is great and no action is needed.

**Comment by feng-4chen** [comment]
> Hi @feng-4chen, I'm not sure I am following. Apologies if I am repeating myself...
> 
> For R: You contributed R/friendman.qmd and Safaraz contributed R/R_Friedmantest.qmd - these need to be merged into one file. For e.g. adding yours as an extra example.
> 
> For SAS: You contributed SAS/friedman.qmd and Safaraz contributed SAS/SAS_Friedmantest.qmd - these need to be merged into one file. For e.g. adding yours as an extra example.
> 
> For SAS vs R: You contributed Comp/r-sas_friedman.qmd and this is the only SAS vs R comparison so this is great and no action is needed.
I will just keep Comp/r-sas_friedman.qmd. The other two is similar content with Safaraz so it' ok. Thanks for explaining!

**Comment by feng-4chen** [comment]
Duplicate SAS/friedman.pmd and R/friedman.qmd removed from branch

**Pull request merged**